spec/mailers/admin_mailer_spec.rb

```diff
@@ -0,0 +1,22 @@
+require "rails_helper"
+
+RSpec.describe AdminMailer, type: :mailer do
+  describe 'send notification email' do
+    let!(:admin) { create(:user, :admin, organization: organization) }
+    let!(:organization) { create(:organization) }
+    let!(:channel) { create(:channel, organization: organization) }
+    let!(:mail) { AdminMailer.delete_channel_email(channel) }
+
+    it 'deliver success' do
```

Comment:
this test is in the wrong place
you testing here NOT mailer
you are testing the model that it's triggering sending a notification email.
Move to the model

AND you don't need to have anything like `let!(:mail) { AdminMailer.delete_channel_email(channel) }` there
JUST channel